<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_title">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>200</width>
      <height>36</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Исходный текст:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_convert">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>280</y>
      <width>150</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>Конвертировать</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="edit_input">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>780</width>
      <height>210</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QTextEdit" name="text_converted">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>380</y>
      <width>780</width>
      <height>210</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_title_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>340</y>
      <width>320</width>
      <height>36</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Конвертированный текст:</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radio_encode">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>270</y>
      <width>121</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Зашифровать</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="radio_decode">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>300</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Расшифровать</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup</string>
    </attribute>
   </widget>
   <widget class="QLabel" name="label_direction">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>280</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Направление</string>
    </property>
   </widget>
   <widget class="QSlider" name="slider_shift">
    <property name="geometry">
     <rect>
      <x>379</x>
      <y>310</y>
      <width>211</width>
      <height>20</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>ClosedHandCursor</cursorShape>
    </property>
    <property name="minimum">
     <number>-20</number>
    </property>
    <property name="maximum">
     <number>20</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_shift">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>280</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Смещение:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_shift_value">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>280</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="button_save_to_file">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>330</y>
      <width>150</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>Сохранить в файл</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
